src/Microsoft.AspNetCore.Mvc.DataAnnotations/ValidationProviderAttribute.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Microsoft.AspNetCore.Mvc.DataAnnotations
+{
+    /// <summary>
+    /// Abstract class for grouping attributes of type <see cref="ValidationAttribute"/> into
+    /// one <see cref="Attribute"/>
+    /// </summary>
+    public abstract class ValidationProviderAttribute : Attribute
+    {
+        /// <summary>
+        /// Gets one or more <see cref="ValidationAttribute" /> instances associated with this attribute.
```

Comment:
It's really `zero or more`, isn't it?

I suggest simplifying to say `Gets <see cref="ValidationAttribute" /> instances associated with this attribute.`

And people can figure it out on their own.

Reply:
Thank you for the review! You're correct, it's zero or more. I'll add your suggestion right away.